step1_get_eia_demand_data.ipynb

```diff
@@ -277,51 +261,38 @@
    "outputs": [],
```

Comment:
Can you update the comment on line 254 above that says "By considering data from 1 Oct 2018 through 30 Sept 2020 we have two full years of data to clean and impute for all except ERCO. For this reason, ERCO will be left as BA-level resolution in the data cleaning and following imputation."? I can't post on that exact line b/c there's no edit there.